### DISCUSSION TITLE
menu组件

### DISCUSSION THREAD

**Discussion by shuailianghong** [ant-design/ant-design#41730] [Q&A]
Antd5 menu组件第一层级没有children的时候class名和样式是否有bug
<img width="168" alt="image" src="https://user-images.githubusercontent.com/87630079/230852827-5b8f589c-d386-485f-8e4b-454c7c9da959.png">

**Comment by MadCcc** [comment]
有最小复现嘛，如果可以复现的话可以直接提 issue
